Description: Bump filter time ven more for launchpad
 .
 fail2ban (0.9.0+git48-gabcab00-3~precise~ppa1) precise; urgency=medium
 .
   Version bump
Author: Rainer Koirikivi <rainer@koirikivi.fi>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- fail2ban-0.9.0+git48-gabcab00.orig/fail2ban/tests/filtertestcase.py
+++ fail2ban-0.9.0+git48-gabcab00/fail2ban/tests/filtertestcase.py
@@ -509,7 +509,7 @@ def get_monitor_failures_testcase(Filter
 			return not self.isFilled(delay)
 
 		def assert_correct_last_attempt(self, failures, count=None):
-			self.assertTrue(self.isFilled(30)) # give Filter a chance to react
+			self.assertTrue(self.isFilled(200)) # give Filter a chance to react
 			_assert_correct_last_attempt(self, self.jail, failures, count=count)
 
 
